x-pack/plugins/data_usage/public/app/components/filters/date_picker.tsx

```diff
@@ -80,9 +88,9 @@ export const UsageMetricsDateRangePicker = memo<UsageMetricsDateRangePickerProps
         showUpdateButton={false}
         timeFormat={'HH:mm'}
         updateButtonProps={{ iconOnly: false, fill: false }}
-        utcOffset={moment().utcOffset() / 60}
-        maxDate={moment()}
-        minDate={moment().subtract(9, 'days').startOf('day')}
+        utcOffset={0}
```

Comment:
at the moment utcOffset doesn't do anything and doesn't set UTC time. We still have to adjust this in code